src/array_api_stubs/_draft/elementwise_functions.py

```diff
@@ -2346,7 +2350,7 @@ def real(x: array, /) -> array:
     Parameters
     ----------
     x: array
-        input array. Should have a complex floating-point data type.
+        input array. Should have a floating-point data type.
```

Comment:
```suggestion
        input array. Should have a numeric data type.
```

Same comment. There is a clearer case here for allowing any numeric data type.